sale_commission/wizard/wizard_settle.py

```diff
@@ -123,3 +120,10 @@ def action_settle(self):
 
         else:
             return {'type': 'ir.actions.act_window_close'}
+
+    def _get_agent_lines(self, agent, date_to_agent):
+        aila = self.env['account.invoice.line.agent'].search([
```

Comment:
Nitpick: could you please avoid this assignment and return directly?

Reply:
I did it